app/uk/gov/hmrc/trackingconsentfrontend/assets/src/common/createNonce.ts

```diff
@@ -1,7 +1,8 @@
-import getNonce from "./getNonce";
+import getNonce from './getNonce';
 
 // The following is required for style-loader to add the nonce attribute
 // correctly to the cookie banner style element
 // It must appear exactly as below (no var, let or const)
 // @ts-ignore
-__webpack_nonce__ = getNonce()
+// eslint-disable-next-line no-underscore-dangle
+window.__webpack_nonce__ = getNonce();
```

Comment:
Has this change been tested? When I implemented it it had to be exactly as it was before for webpack to pick it up, hence the @ts-ignore.

Reply:
Fair question.  I haven't, let me see if I can ignore that line instead.

Reply:
Updated to avoid changing the production code.